patches/expose_flags.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
+index ff1ed1a98cbfe..7c02d25eef176 100644
+--- a/chrome/browser/about_flags.cc
++++ b/chrome/browser/about_flags.cc
+@@ -4117,6 +4117,10 @@ const FeatureEntry kFeatureEntries[] = {
+ // //tools/flags/generate_unexpire_flags.py.
+ #include "build/chromeos_buildflags.h"
+ #include "chrome/browser/unexpire_flags_gen.inc"
++    {"disable-cross-origin-referrers",
++     flag_descriptions::kDisableCrossOriginReferrersName,
++     flag_descriptions::kDisableCrossOriginReferrersDescription, kOsAll,
++     FEATURE_VALUE_TYPE(switches::kDisableCrossOriginReferrers)},
```

Comment:
it seems like it would make more sense to have a single flag for this, otherwise there will be combinations where the logic doesn't make sense, like:

kDisableCrossOriginReferrers: enabled,
kCapReferrerToOriginOnCrossOrigin: disabled

Instead if we had one flag with these options:

kCrossOriginReferrerSetting:

- "Disable cross origin referrers"
- "Cap referrer to origin on cross origin (default)"
- "Unrestricted referrer on cross origin"

Reply:
@RKNF404 makes sense, let's keep it as-is at least for now